public/alcom/releases/index.hbs

```diff
@@ -27,25 +27,42 @@ with="social"
 <h2 class="inline-block u-no-margin--bottom me-1">{{common.releases}}</h2>
 <aside class="p-accordion mt-2">
     <ul class="p-accordion__list" id="releases-list">
-        {{#each __releases}}
+        {{#each data.releases}}
         {{#if isGui}}
         <li class="p-accordion__group">
             <h4 role="heading" aria-level="3" class="p-accordion__heading">
                 <button type="button" class="p-accordion__tab" id="header-{{id}}" aria-controls="section-{{id}}"
                     aria-expanded="false">{{displayName}}</button>
             </h4>
             <section class="p-accordion__panel" id="section-{{id}}" aria-hidden="true" aria-labelledby="header-{{id}}">
-                {{#each downloads}}
-                <hr class="is-muted mb-1" />
-                <div class="flex mb-1">
-                    <div class="flex-grow">
-                        <code>{{name}}</code>
+                <div class="p-card u-no-padding mt-1">
+                    <div class="p-card__inner">
+                        {{renderMarkdown body}}
                     </div>
-                    <div class="flex-shrink">
-                        <a class="color-link" href="{{link}}">{{../../common.download}}</a>
+                    <hr class="u-no-margin--bottom">
+                    <div class="p-card__inner">
+                        <a class="p-button has-icon mb-0" href="{{html_url}}">
+                            <i class="p-icon--external-link"></i>
+                            <span>{{../releases.viewOnGitHub}}</span>
+                        </a>
                     </div>
                 </div>
-                {{/each}}
+                <div class="p-card pt-1 pb-1">
+                    <h4>{{../releases.assets}}</h4>
+                    {{#each assets}}
+                    {{#if (not @first)}}
+                    <hr class="is-muted mb-1" />
+                    {{/if}}
+                    <div class="flex mb-1">
+                        <div class="flex-grow">
+                            <code>{{name}}</code>
+                        </div>
+                        <div class="flex-shrink">
+                            <a class="color-link" href="{{browser_download_url}}">{{../../common.download}}</a>
+                        </div>
+                    </div>
+                    {{/each}}
+                </div>
```

Comment:
It might be better to merge two cards onto one. since all releases have heading, it's a little confusing to spit to two cards. 

I think releasenote, asset list and link to github in this order in one card is good.